packages/core/src/components/DatePicker/DatePicker.stories.tsx

```diff
@@ -20,14 +20,40 @@ import {
   theme,
 } from "@hitachivantara/uikit-react-core";
 
-const Decorator = ({ children }) => {
-  return <div style={{ width: 340, height: 600, padding: 10 }}>{children}</div>;
-};
+const overflowDecorator: DecoratorFn = (Story) => (
```

Comment:
We probably should find a place to put some "story utilities", that's easily accessible from all packages